stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
+variables. The variables can be either continuous or discontinuous, which are
+further divided into finite and infinite. They are associated
+with probability densities on corresponding domains and internally defined
+in terms of probability spaces. 
+Apart from the possibility of defining the random variables from user supplied
+density distribution, SymPy provides definitions of most common
+distributions, including \texttt{Uniform}, \texttt{Poisson}, \texttt{Normal},
+\texttt{Binomial}, \texttt{Bernoulli}, and many others.
+
+Properties of random expressions can be calculated using, e.g.,
```

Comment:
Expectation, variance, and covariance map random expressions to real numbers. I wouldn't call that properties.